src/GovUk.Education.ExploreEducationStatistics.Content.Services/DataSetFileService.cs

```diff
@@ -59,11 +60,14 @@ public class DataSetFileService : IDataSetFileService
             _contentDbContext.ReleaseVersions.LatestReleaseVersions(publicationId, publishedOnly: true);
 
         var query = _contentDbContext.ReleaseFiles
+            .AsNoTracking()
+            .Include(rf => rf.File)
             .OfFileType(FileType.Data)
             .HavingNoDataReplacementInProgress()
             .HavingThemeId(themeId)
             .HavingPublicationIdOrNoSupersededPublication(publicationId)
             .HavingReleaseVersionId(releaseVersionId)
+            .HavingApiDataSet(apiDataSetsOnly ?? false)
```

Comment:
Doesn't really matter, but I think you could use `apiDataSetsOnly.Value` here, like `latestOnly.Value` below?